blog/_posts/2016-01-21-compiler-hacking-in-london.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: post
+title:  Twelfth OCaml compiler hacking evening in London
+date:   2016-01-13 12:00:00
```

Comment:
This probably needs to be `2016-01-14`, although I'm not sure how much it matters.  I think it's used in RSS and Atom feeds.